optional/radicale/Dockerfile

```diff
@@ -1,7 +1,9 @@
-FROM alpine:edge
+ARG EDGE_DISTRO=alpine:edge
+FROM $EDGE_DISTRO
 
```

Comment:
Edge is no longer used:
https://github.com/Mailu/Mailu/blob/13d6cf2fdbcf9b4e3bd31d5f8db467ee340cbc74/optional/radicale/Dockerfile#L1